manifests/repos/apt.pp

```diff
@@ -34,7 +34,12 @@ # == class mongodb::repos::apt
       default  => 'http://downloads-distro.mongodb.org/repo/debian-sysvinit',
     }
 
-    $package_name = 'mongodb-10gen'
+    if ($mongodb::package_name == undef) {
```

Comment:
package_name is set to 'mongodb-10gen' by params.pp. So if you set it to undef it will be the same as the default.